backend/canisters/user/impl/src/updates/mark_read_v2.rs

```diff
@@ -29,6 +30,13 @@ fn mark_read_impl(args: Args, runtime_state: &mut RuntimeState) -> Response {
                     .threads_read
                     .insert(thread.root_message_index, thread.read_up_to, now);
             }
+
+            if chat_messages_read.date_read_pinned.is_some() {
```

Comment:
Maybe this?

`if chat_messages_read.date_read_pinned > group_chat.date_Read_pinned.value {
...
}`

Reply:
Yes I was just reviewing this